test/Fondue.test.js

```diff
@@ -342,45 +346,116 @@ describe("Layout features", () => {
 	test("returns lookup 1 layout features", async () => {
 		const fondue = await SourceCodeProOTFFont();
 
-		expect(fondue.featureChars["DFLT"]["dflt"]).toEqual(
+		expect(
+			fondue.featureChars["DFLT"]["dflt"]["onum"]["lookups"][0]
+		).toEqual(
 			expect.objectContaining({
-				onum: {
-					type: 1,
-					input: ["0", "1", "2", "3", "4", "5", "6", "7", "8", "9"],
-					alternateCount: [],
-				},
+				type: 1,
+				typeName: "Single Substitution",
+				lookahead: [],
+				backtrack: [],
+				input: ["0", "1", "2", "3", "4", "5", "6", "7", "8", "9"],
+				alternateCount: [],
 			})
 		);
 	});
 
 	test("returns lookup 3 layout features", async () => {
 		const fondue = await AthenaRubyFont();
 
-		expect(fondue.featureChars["DFLT"]["dflt"]).toEqual(
+		expect(
+			fondue.featureChars["DFLT"]["dflt"]["cv01"]["lookups"][0]
+		).toEqual(
 			expect.objectContaining({
-				cv01: {
-					type: 3,
-					input: ["Α"], // Note this is U+0391 : GREEK CAPITAL LETTER ALPHA
-					alternateCount: [27],
-				},
+				type: 3,
+				typeName: "Alternate Substitution",
+				lookahead: [],
+				backtrack: [],
+				input: ["Α"], // Note this is U+0391 : GREEK CAPITAL LETTER ALPHA
+				alternateCount: [27],
 			})
 		);
 	});
 
 	test("returns lookup 4 layout features", async () => {
 		const fondue = await OpenSansFont();
 
-		expect(fondue.featureChars["latn"]["dflt"]).toEqual(
+		expect(
+			fondue.featureChars["latn"]["dflt"]["liga"]["lookups"][0]
+		).toEqual(
 			expect.objectContaining({
-				liga: {
-					alternateCount: [],
-					input: ["ffl", "ffi", "ff", "fl", "fi"],
-					type: 4,
-				},
+				type: 4,
+				typeName: "Ligature Substitution",
+				lookahead: [],
+				backtrack: [],
+				input: ["ffl", "ffi", "ff", "fl", "fi"],
+				alternateCount: [],
 			})
 		);
 	});
 
+	test("returns lookup 6 layout features", async () => {
+		const fondue = await AbrilFatFaceFont();
+
+		expect(
+			// Yes, four empty spaces (artifact of old font editing software)
+			fondue.featureChars["    "]["dflt"]["calt"]["lookups"][0]
+		).toEqual({
+			alternateCount: [],
+			backtrack: [
+				["j"],
```

Comment:
Can this return anything other than one character? If not, this doesn't have to be an array. If it can, this is missing some test coverage?